obspy/taup/seismic_phase.py

```diff
@@ -53,8 +57,10 @@ class SeismicPhase(object):
     # Note this is not the total distance, only the segment along the CMB.
     # The default is 60 degrees.
     maxDiffraction = 60
+    # The maximum number of refinements to make to an Arrival.
+    maxRecursion = 5
 
-    def __init__(self, name, tMod):
+    def __init__(self, name, tMod, receiver_depth=0.0):
```

Comment:
Is it really necessary this deep in? I think most people would use `TauPyModel`, not `SeismicPhase` directly.